L/libpolymake_julia/build_tarballs.jl

```diff
@@ -14,7 +14,6 @@ version = v"0.8.0"
 
 julia_versions = [v"1.6.3", v"1.7.0", v"1.8.0", v"1.9.0"]
```

Comment:
I've been thinking that for all these packages we should propagate this set of versions into the julia compat, which should be something like
```julia
julia> join("~" .* string.(julia_versions), ", ")
"~1.6.3, ~1.7.0, ~1.8.0, ~1.9.0"
```
or
```julia
julia> join("~" .* string.(getfield.(julia_versions, :major)) .* "." .* string.(getfield.(julia_versions, :minor)), ", ")
"~1.6, ~1.7, ~1.8, ~1.9"
```
if we want to allow the entire minor series (for example important for v1.6).  This would allow us to stop building for 1.7, 1.8 in the future when those series will not be maintained anymore.

Reply:
done, including a reminder to update the version when restricting the versions.

Reply:
Good point, thanks! :+1: